home/modules/ROOT/pages/mobile.adoc

```diff
@@ -1,69 +1,91 @@
-= Couchbase Mobile - Embedded Database on the Edge
-:page-layout: landing-page-top-level-sdk
-:page-role: tiles
+= Couchbase Mobile 
+// :page-layout: landing-page-top-level-sdk
+// :page-role: tiles 
+:page-toclevels: 2
 :page-rank: 75
-:!sectids:
 
-include::partial$info-banner.adoc[]
+Couchbase Mobile is a comprehensive NoSQL database solution for building offline-first mobile and edge applications. 
+It enables seamless data synchronization between devices and the cloud, with automatic conflict resolution and security.
```

Comment:
Review with PM - we're losing a few features/keywords with these changes in general, which feels like they might've been here for SEO reasons?

- `SQL++`
- `NoSQL JSON Document Style database`
  - I don't know whether this is important but it seemed like very specific phrasing so worth checking
- `full-text search`